randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -1652,9 +1661,7 @@ Extra - asset_id: collision_camera_019
   > Event - Storm Missile Gate
       Activate Storm Missile Locks
   > Bomb Ledge
-      All of the following:
-          Screw Attack
-          Walljump (Beginner) or Use Spin Boost
+      Screw Attack
```

Comment:
In that case, this one should also account for that cause you have to use a ledge grab iirc

Reply:
No you don't need a ledge grab there. You can just jump up